docs/fundamentals/app-lifecycle.md

```diff
@@ -0,0 +1,440 @@
+---
+title: "App lifecycle"
+description: ".NET MAUI raises cross-platform lifecycle events when an app transitions between its different execution states."
+ms.date: 09/29/2021
+---
+
+# App lifecycle
+
+.NET Multi-platform App UI (.NET MAUI) apps generally have three execution states: *not running*, *running*, and *stopped*. .NET MAUI raises lifecycle cross-platform events on the `Window` class when an app transitions from the not running state to the running state, and the running state to the stopped state.
```

Comment:
Referring to my comment about the picture I'd add *deactivated* to this list of states. For example, on android when a permission dialog opens up the app is *deactivated* but it's not stopped.